CocktailRecipes/CocktailRecipes/Features/Home/Charts/DrinkListPresenter.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  DrinkListPresenter.swift
+//  CocktailRecipes
+//
+//  Created by Jaison Vieira on 10/17/20.
+//  Copyright © 2020 Eric Rado. All rights reserved.
+//
+
+import Foundation
+
+struct DrinkListPresenter {
```

Comment:
I have been using classes for presenters

Reply:
Done.